File: backend/src/main/java/de/grimsi/gameyfin/GameyfinApplication.java
package de.grimsi.gameyfin;

import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@SpringBootApplication
@ConfigurationPropertiesScan
public class GameyfinApplication {

    public static void main(String[] args) {
        new SpringApplicationBuilder(GameyfinApplication.class)
                .properties( "file.encoding=UTF-8", "spring.config.name=application,gameyfin,database,secure")
                .build()
                .run(args);
    }

}

File: src/main/kotlin/de/grimsi/gameyfin/GameyfinApplication.kt
package de.grimsi.gameyfin

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.transaction.annotation.EnableTransactionManagement


@SpringBootApplication
@EnableTransactionManagement
class GameyfinApplication

fun main(args: Array<String>) {
    runApplication<GameyfinApplication>(*args)
}